src/services/expensesServices.js

```diff
@@ -0,0 +1,47 @@
+'use strict';
+
+const { unicId } = require('../helpers/helpers');

```

Comment:
There seems to be a typo in the imported function name. It should be 'uniqueId' instead of 'unicId'.